### ISSUE TITLE
Select points by range

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#52]
Gus wants a button he can use to select with one click all the demagnetization steps from 100 to 600 mT. Obviously this functionality should be implemented in a slightly more general fashion...

**Comment by pont-us** [comment]
```
Bugs Everywhere data:
Created at: 2009-04-13 21:19:41
Status: open
Severity: minor
UUID: 085dc9c9-c0d8-4233-8446-5ce4dd65d07c
Short name: a88/085
Ditz bug index: 50
```

**Comment by pont-us** [comment]
Closed by fa3b4110cc4fc433eb3fc8dc1e4228b1411600f5.
